Sources/GitBuddyCore/Helpers/DateFormatters.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DateFormatters.swift
+//  
+//
+//  Created by Antoine van der Lee on 25/01/2022.
```

Comment:
nit: shouldn't this contain the copyright notice like on https://github.com/WeTransfer/GitBuddy/blob/master/Sources/GitBuddy/main.swift, or just not be here at all?

Reply:
Good one!